fastlane/Fastfile

```
@@ -0,0 +1,48 @@
+     required_env_vars = ['FIREBASE_APP_ID']
+     required_env_vars.each do |var|
+         UI.user_error!("Missing required environment variable: #{var}") if ENV[var].nil?
+     end
+
+     latest_release = firebase_app_distribution_get_latest_release(
+        app: ENV['FIREBASE_APP_ID'],
+        service_credentials_file: "firebase_credentials.json"
+        ) rescue begin
+               UI.important("Failed to fetch latest release, defaulting to version 1")
+               OpenStruct.new(buildVersion: "0")
+     )
```

RuboCop (1.69.1)

[fatal] unexpected token tRPAREN
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)